src/main/scala/scoverage/ScoverageSbtPlugin.scala

```diff
@@ -255,8 +294,9 @@ object ScoverageSbtPlugin extends AutoPlugin {
 
         CoverageMinimum.all.value
           .checkCoverage(cov, coverageFailOnMinimum.value)
+
       case None =>
-        log.info("No subproject data to aggregate, skipping reports")
+        log.warn(s"No coverage data found to aggregate in [$target]. Run >sbt coverage test< first.")
```

Comment:
Turning it into a warning (because the other log messages are warnings) and making it more expressive